.spelling

```diff
@@ -273,6 +274,7 @@ JViewer
 Kafka
 Kea
 Keepalived
+k8s
```

Comment:
In documentation, I think we should spell out kubernetes instead of adding k8s to the dictionary.

Reply:
Fixed in commit 3c62b31.